TipulDay.py

```diff
@@ -1,178 +1,182 @@
 
-print("Started Submitting Forms.")
+# Code that inputs the form into all the websites
 
-if(len(websites)>0):
+print("[*] Submitting forms")
 
-    SuccessfullForms=0
-    FailedForms=0
+if(len(websites) > 0):
+    SuccessfullForms = 0
+    FailedForms = 0
     for website in websites:
-        #specifying the website url
+        # specifying the website url
         url = website
 
-        #making a request to the website and getting the response 
+        # making a request to the website and getting the response
         try:
             response = requests.get(url)
         except:
-            print("Error 5: Issue requesting from the website.")
-            FailedForms=FailedForms+1
+            print("[!] Error 5: Issue requesting from the website")
+            FailedForms += 1
             continue
 
-        #parsing the response using BeautifulSoup
+        # parsing the response using BeautifulSoup
         soup = BeautifulSoup(response.text, 'html.parser')
 
-        #finding the form elements in the page 
+        # finding the form elements in the page
         form_elements = soup.find_all('input')
         #form_elements.append(soup.find_all('input', {'type': 'tel'}))
-        #creating a dictionary to store the form data
+        # creating a dictionary to store the form data
         form_data = {}
 
-        #looping through the form elements and getting the data 
+        # looping through the form elements and getting the data
         for element in form_elements:
-                name = element.get('name')
-                value = element.get('value')
-                form_data[name] = value
+            name = element.get('name')
+            value = element.get('value')  # why is the value necessary?
+            form_data[name] = value
 
-    
         for data in form_data:
-                    DataName =data
-                    try:
-                        if "name" in DataName:
-                                form_data[DataName] = FormNameValue
-                        if "phone" in DataName or "num" in DataName or "tel" in DataName:
-                                form_data[DataName] = FormPhoneNumberValue  
-                        if "mail" in DataName:
-                                form_data[DataName] = FormEmailValue   
-                        if "Comment"in DataName or "message"in DataName: 
-                                form_data[DataName] = FormCommentValue  
-                    except:
-                        print('Error 6: Some parameters might be missing:',website,'.')
-                         
-
-
-                #making a post request to the website with the form data 
+            DataName = data
+            try:
+                if "name" in DataName:  # Some developers like to name the fields fname or lname so this will miss
```

Comment:
this is not true becuase " if "name" in DataName: " will search the string "name" in the main string, it will find fname+lname

Reply:
Checkout this example
![Untitled](https://user-images.githubusercontent.com/94757536/206928070-089f9748-7a29-43d4-a0de-f994c7627ba8.png)

Reply:
![Untitled](https://user-images.githubusercontent.com/69360460/206931210-17e8a0e3-36f6-48b0-906c-61aa6cc3ccbc.png)

it works this way in the code

Reply:
Oh my fault i thought you were addressing an array and not a string. Lemme check again